prep-css-syntax/index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>CSS Syntax</title>
+</head>
+<body>
+  <h1 style="font-family: Arial, Helvetica, sans-serif;">This h1 element has been given Arial as a font-family.</h1>
+  <p style="font-size: 2 rem;">I am a p element which has a font-size of 2rem!!!</p>
+  <h3 style="color: firebrick;">This h3 element has a font color of firebrick.</h3>
+  <h2 style="font-style: italic;">This h2 element has an italic font-style.</h2>
```

Comment:
Hi Frank, for this exercise we want to use a `<style>` element in the `<head>` instead of `style` attributes.